scripts/maint/practracker/exceptions.txt

```diff
@@ -33,7 +33,7 @@
 #
 # Remember: It is better to fix the problem than to add a new exception!
 
-problem file-size /src/app/config/config.c 7200
+problem file-size /src/app/config/config.c 7400
```

Comment:
Can you put some of these newly split functions in another file?

Reply:
Yes, but I would like to do that later, since the next step of refactoring will be to make options_act_reversible behave more sensibly, in https://trac.torproject.org/projects/tor/ticket/32408.